tests/src/core/testqgsgraduatedsymbolrenderer.cpp

```diff
@@ -136,5 +141,29 @@ void TestQgsGraduatedSymbolRenderer::rangesHaveGaps()
   QVERIFY( renderer.rangesHaveGaps() );
 }
 
+void TestQgsGraduatedSymbolRenderer::_makeBreaksSymmetric(QList<double> &breaks, double symmetryPoint, bool astride)
+{
+  const QList<double> unchanged_breaks = {1235, 1023, 997, 800, 555, 10, 1, -5, -11, -423, -811};
+  
+  // with astride = false
+  breaks = unchanged_breaks;
+  symmetryPoint = 12.0;
+  astride = false;
+  _makeBreaksSymmetric( breaks, symmetryPoint, astride );
```

Comment:
I'm a bit puzzled by this call. Will this not create an infinite recursion on this method?

Reply:
I don't understand how I am supposed to apply my function then.
I tried lot of things, its the only version that compiles. 
Am I supposed to declare my parameters in init() ? Maybe with a pointer so that I can access them after without parameters ?

Reply:
I guess you want a test function (without parameters) that calls this function with appropriate parameters?

Reply:
I don't know. Something that works will be enough.
Maybe I am obliged to make _makeBreaksSymmetric() a class function ?